test/objects.spec.js

```diff
@@ -181,10 +182,9 @@ describe('Object Functions', () => {
     expect(f.compareDeep('a.b.c')(o)('1')).to.deep.equal(false)
   })
   it('mapProp', () => {
-    const a = f.mapProp(val => val * val, 'a', { a: 2, b: 1 })
+    const a = _.update(val => val * val, 'a', { a: 2, b: 1 })
```

Comment:
Sorry for the additional feedback, but I think you may have misunderstood my point 😄 This test should probably not be modified at all in the PR since we're not changing the behavior - and also, if the unit test uses lodash instead of the futil method, then we have no way of knowing if F.mapProp works.